src/web_app_skeleton/script/setup.ecr

```diff
@@ -10,6 +10,20 @@ indent() {
   done
 }
 
+# Ensure postgres is installed, and booted
+ensure_postgres() {
+  if ! command -v createdb > /dev/null; then
```

Comment:
This is a bit tricky because this actually is checking for the CLI tools. You may have Postgres installing, but not the CLI tools. So we should probably be more specific and say Postgres CLI tools are not installed and say if you're using Postges.app to https://postgresapp.com/documentation/cli-tools.html

And then a separate check for `psql`  or  something that  will check if Postgres itself is installed. That should probably go before the `created` check. Does that make sense?

Reply:
is `psql` a part of the cli tools? I know there's also a `postgres` command for the server. But maybe you don't have that one if you only installed the pg client?

Reply:
Yup https://www.postgresql.org/docs/11/reference-client.html Looks like those fall under pg client. I guess you don't technically have to have the postgres server installed locally because you could be connecting to a remote DB. And all of the client apps allow you to run them locally and still connect remotely.

I'll update the wording, but I think I'll skip the extra check.

Reply:
Sounds good!